devices/iluminize/led_controller_multi_5.cct.json

```diff
@@ -0,0 +1,186 @@
+{
+  "schema": "devcap1.schema.json",
+  "manufacturername": "iluminize",
+  "modelid": "HK-ZD-CCT-A",
+  "product": "LED Controller Multi 5 (CCT mode)",
+  "sleeper": false,
+  "status": "Gold",
+  "subdevices": [
+    {
+      "type": "$TYPE_COLOR_TEMPERATURE_LIGHT",
+      "restapi": "/lights",
+      "uuid": [
+        "$address.ext",
+        "0x01"
+      ],
+      "items": [
+        {
+          "name": "attr/id"
+        },
+        {
+          "name": "attr/lastannounced"
+        },
+        {
+          "name": "attr/lastseen"
+        },
+        {
+          "name": "attr/manufacturername"
+        },
+        {
+          "name": "attr/modelid"
+        },
+        {
+          "name": "attr/name"
+        },
+        {
+          "name": "attr/swversion"
+        },
+        {
+          "name": "attr/type"
+        },
+        {
+          "name": "attr/uniqueid"
+        },
+        {
+          "name": "cap/color/capabilities",
+          "refresh.interval": 86400
+        },
+        {
+          "name": "cap/color/ct/max",
+          "refresh.interval": 86400,
+          "default": "0xfeff"
+        },
+        {
+          "name": "cap/color/ct/min",
+          "refresh.interval": 86400,
+          "default": 0
+        },
+        {
+          "name": "cap/color/xy/blue_x",
+          "static": 0
+        },
+        {
+          "name": "cap/color/xy/blue_y",
+          "static": 0
+        },
+        {
+          "name": "cap/color/xy/green_x",
+          "static": 0
+        },
+        {
+          "name": "cap/color/xy/green_y",
+          "static": 0
+        },
+        {
+          "name": "cap/color/xy/red_x",
+          "static": 0
+        },
+        {
+          "name": "cap/color/xy/red_y",
+          "static": 0
+        },
+        {
+          "name": "config/bri/couple_ct",
+          "default": true
+        },
+        {
+          "name": "config/bri/on_level",
+          "refresh.interval": 3600,
+          "default": 255
+        },
+        {
+          "name": "config/bri/onoff_transitiontime",
+          "refresh.interval": 3600,
+          "default": 1
+        },
+        {
+          "name": "config/bri/startup",
+          "refresh.interval": 3600,
+          "default": 255
+        },
+        {
+          "name": "config/color/ct/startup",
+          "refresh.interval": 3600,
+          "default": 336
+        },
+        {
+          "name": "config/color/execute_if_off",
+          "refresh.interval": 3600,
+          "default": true
+        },
+        {
+          "name": "config/color/xy/startup_x",
+          "refresh.interval": 3600,
+          "default": "0xffff"
+        },
+        {
+          "name": "config/color/xy/startup_y",
```

Comment:
These are only supported on Hue lights.

Reply:
Added what?  HomeKit doesn’t have poweron settings, and certainly not for colour.  I’ve seen some vendors that (illegally) expose the Hue _Manufacturer Code_ 0x100B in the _Node Descriptor_ (also missing from the screenshots), so the Hue bridge exposes them to HomeKit.  In that case, they might expose the Hue-specific attributes for poweron xy as well.  However, they are not in the screenshots?  Do you see them in the GUI?  And do they work?